src/Microsoft.AspNet.Identity.AspNetCoreCompat/IdentityDbContext.cs

```diff
@@ -126,16 +126,15 @@ protected override DbEntityValidationResult ValidateEntity(DbEntityEntry entityE
         {
             if (entityEntry.Entity is TUser && entityEntry.State == EntityState.Modified)
             {
-                entityEntry.Property("ConcurrencyToken").CurrentValue = Guid.NewGuid();
+                entityEntry.Property("ConcurrencyStamp").CurrentValue = Guid.NewGuid().ToString();
```

Comment:
FYI, if you cast the `DbEntityEntry` to a generic `DbEntityEntry<TEntity>`  for the right entity type I think you should be able to use the lambda syntax. It is fine to keep it like this though.